ERROR:root:'interests'
Traceback (most recent call last):
  File "C:\Users\vikto\Documents\Old computer files\SFA\Scramble_for_Africa.py", line 991, in <module>
    main_loop.main_loop(global_manager)
  File "C:\Users\vikto\Documents\Old computer files\SFA\modules\main_loop.py", line 42, in main_loop
    current_button.on_click()
  File "C:\Users\vikto\Documents\Old computer files\SFA\modules\buttons.py", line 775, in on_click
    self.global_manager.get('save_load_manager').load_game('save1.pickle')
  File "C:\Users\vikto\Documents\Old computer files\SFA\modules\save_load_tools.py", line 354, in load_game
    self.global_manager.get('actor_creation_manager').load_minister(current_minister_dict, self.global_manager)
  File "C:\Users\vikto\Documents\Old computer files\SFA\modules\actor_creation_tools.py", line 238, in load_minister
    new_minister = ministers.minister(True, input_dict, global_manager)
  File "C:\Users\vikto\Documents\Old computer files\SFA\modules\ministers.py", line 55, in __init__
    self.interests = input_dict['interests']
KeyError: 'interests'
